README.md

```diff
@@ -32,7 +32,18 @@ To build for the EV3:
 
 * On your host computer (not in `brickstrap shell`), use NFS or sshfs to share
 <some-path-like-home/user/build-brickman> with your EV3.
-* On your EV3, connect the share and run `./brickman`
```

Comment:
Should we have some devscript repo? Or have this on the website under http://www.ev3dev.org/docs/devtools/. Personally I find it easy enought to store the service, update the file and restart the service.